be/src/util/sm3.cpp

```diff
@@ -8,6 +8,7 @@ bool Sm3::is_little_endian = (*(char*)&Sm3::ENDIAN_TEST_VALUE == 1) ? true : fal
 
 // Cyclic shift left k-bit operation
 unsigned int Sm3::left_rotate(unsigned int word, int k) {
+    k = k % WORD_BITS;
```

Comment:
If I is greater than WORD BITS it should be it other than other number?

Reply:
if k is greater or equal to the width of type 'unsigned int', the result of the right shift is undefined.